src/components/Input/Input.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import Icon from 'components/Icon';
+import { useSelector } from 'react-redux';
```

Comment:
You don't like where it was? Just curious, no need to revert the change.

Reply:
haha this one was an accident, I sometimes instinctively hit <kbd>Shift</kbd> + <kbd>⌘</kbd> + <kbd>O</kbd> which organizes the imports, let me revert